compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParser.java

```diff
@@ -11332,7 +11332,7 @@ private boolean isEndOfAnonFuncParametersList(SyntaxKind tokenKind) {
      */
     private STNode parseTupleTypeDesc() {
         STNode openBracket = parseOpenBracket();
-        startContext(ParserRuleContext.TYPE_DESC_IN_TUPLE);
+        startContext(ParserRuleContext.TUPLE_MEMBER_OPTIONAL_ANNOTATIONS);
```

Comment:
Well, we are using this context to parse a list of tuple members. So the context name is misleading. 
Shall we have something like `TUPLE_MEMBERS`.